tests/main/retry-network/task.yaml

```
@@ -24,6 +24,13 @@ debug: |
 execute: |
     echo "Add totally unconnected network namespace"
     ip netns add testns
+
+    # Without having "lo" the ipv6 test will retrun EADDRNOTAVAIL which
```

Is this what you meant?
```suggestion
    # Without having "lo" up the ipv6 test will retrun EADDRNOTAVAIL which
```